Comments/ClientApp/src/components/Review/Review.js

```diff
@@ -45,7 +46,7 @@ type PropsType = {
 type StateType = {
 	consultationData: ConsultationDataType | null,
 	commentsData: ReviewPageViewModelType | null,
-	userHasSubmitted: boolean,
+	submittedDate: boolean,
```

Comment:
Should this be of type Date?

Reply:
Has been changed to Date as per suggestion.